### PR TITLE
es-consul-discovery plugin for es-2.2.2

### PR DESCRIPTION
**Pull request by ghostik**
In this PR created deb package es-consul-discovery plugin for es-2.2.2.
@mkostrikin please, review.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -13,4 +13,7 @@
 
 # Maven
 log/
-target/
\ No newline at end of file
+target/
+
+#Packages
+*.deb
```
Comment (ashilokhvostov) [inline .gitignore]: No new line

File: src/deb/control/control [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Source: elasticsearch-consul-discovery
+Section: java
+Priority: extra
+Version: [[version]]
+Maintainer: Konatsntin Tushakov <ktu@tradeshift.com>
```
Comment (mkostrikin) [inline src/deb/control/control]: `Konatsntin`

**Comment by ashilokhvostov** [comment]
:+1:

**Comment by mkostrikin** [comment]
:floppy_disk: unless comments/questions above

**Comment by mkostrikin** [comment]
please squash commits

**Comment by diogokiss** [comment]
💋

**Pull request merged**